packages/shared/src/api/channelsApi.ts

```diff
@@ -303,3 +341,63 @@ function toThreadUnreadStateData(unread: ub.Unread): db.ThreadUnreadState[] {
     };
   });
 }
+
+export function createPendingMultiDmChannel(
```

Comment:
I think this and `createSingleDmChannel` should either be named differently or go in a different file -- it's a little confusing that they're named similarly to `createDefaultChannel` but don't actually mutate anything on the backend.